### ISSUE TITLE
Internal keyboard does not work in firmware after warm boot.

### ISSUE THREAD

**Issue by johanes2115** [Dasharo/dasharo-issues#445]
**Dasharo version**
Dasharo Enterprise 0.9.0

**Dasharo variant**
nv4x_tgl

**Affected component(s) or functionality**
Keyboard

**Brief summary**
Internal keyboard does not work in firmware after warm boot.

**How reproducible**
100%

**How to reproduce**

1. Power on the DUT.
2. Log into the OS.
3. Warm boot the DUT
4. Try to enter firmware using assigned key.

**Expected behavior**
The DUT should enter into the firmware.

**Actual behavior**
The DUT does not react to keyboard key presses in the firmware.

**Screenshots**

**Additional context**

**Solutions you've tried**

**Comment by miczyg1** [comment]
Duplicate of https://github.com/Dasharo/dasharo-issues/issues/295 ?

**Comment by johanes2115** [comment]
> Duplicate of #295 ?

I discovered the issue only after performing warm boot and it's 100% reproducible. If the case is sufficiently similar to #295 in your opinion, I can put my observations in the comment to that issue.

**Comment by miczyg1** [comment]
Main question is if the same issue is 100% reproducible on standard Dasharo for NV4X TGL. If you can check it, then I will be able t omake a decision whether it is a duplicate or not.

**Comment by johanes2115** [comment]
The problem is 100% reproducible on standard Dasharo for NV4X TGL, version 1.4.0.

**Comment by miczyg1** [comment]
Also why it hasn't been detected during the v1.4.0 release validation? I don't see it in known issues: https://docs.dasharo.com/variants/novacustom_nv4x_tgl/releases/
